testing/testing.go

```diff
@@ -81,7 +90,7 @@ func EnsureNodeSync(nodeManager common.NodeManager) error {
 
 	timeouter := time.NewTimer(20 * time.Minute)
 	defer timeouter.Stop()
-	ticker := time.NewTicker(time.Second)
+	ticker := time.NewTicker(1 * time.Second)
```

Comment:
Isn't it the same? :) What's the point in `1 *`?

Reply:
It is the same. But I liked it, it's more clear in separation of quantity and unit.

Reply:
It's the same, just a matter of style, I like it to be consistent.